src/media/css/tiles.styl

```diff
@@ -71,6 +71,10 @@ $greater-than-mobile = unquote('(min-width: 330px)');
         margin: 0 auto;
         width: 50%;
     }
+    +website-detail--wide() {
```

Comment:
I don't particularly mind app default with website override for now. It's something we can always refactor later if we get serious about websites.